assets/src/edit-story/components/canvas/pagemenu/index.js

```diff
@@ -52,7 +52,7 @@ const Wrapper = styled.div`
 `;
 
 const Box = styled.div`
-  background-color: ${({ theme }) => theme.colors.bg.v1};
+  background-color: transparent;
```

Comment:
This is for displaying the frame through the page actions but not being on top of it.